rt_utils/rtstruct.py

```diff
@@ -106,7 +106,7 @@ def get_roi_mask_by_name(self, name) -> np.ndarray:
         """
 
         for structure_roi in self.ds.StructureSetROISequence:
-            if structure_roi.ROIName == name:
+            if structure_roi.ROIName.casefold() == name.casefold():
```

Comment:
```python
    def get_roi_mask_by_name(self, name: str) -> np.ndarray:
        """
        Returns the 3D binary mask of the ROI with the given input name
        """
        name = name.casefold() if type(name) is str else name

        for structure_roi in self.ds.StructureSetROISequence:
            if structure_roi.ROIName.casefold() == name:
                contour_sequence = ds_helper.get_contour_sequence_by_roi_number(
                    self.ds, structure_roi.ROINumber
                )
                return image_helper.create_series_mask_from_contour_sequence(
                    self.series_data, contour_sequence
                )

        raise RTStruct.ROIException(f"ROI of name `{name}` does not exist in RTStruct")
```

It may be safer to check that `name` is not none before calling `.casefold()` as shown above. Thoughts @asim-shrestha ?